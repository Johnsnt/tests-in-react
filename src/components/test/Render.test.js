import React from 'react';
import { render } from '@testing-library/react';
import Render from './../Render';



describe("Snapshot test", () => {

 
 it('should take a snapshot', () => {
  //desestrutura um objeto retornado de render que retorna um fragmento da redenrização
  const { asFragment } = render(<Render />);
  expect(asFragment(<Render />)).toMatchSnapshot();
 });


 it('take a snapshot test (render using other sintax)', () => {
  const renderElement = render(<Render/>)
  expect(renderElement.asFragment()).toMatchSnapshot();

 });

});