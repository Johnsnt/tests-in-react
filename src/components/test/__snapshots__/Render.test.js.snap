// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render tests render using other sintax 1`] = `
<DocumentFragment>
  <div>
    Render
  </div>
</DocumentFragment>
`;

exports[`Render tests should take a snapshot 1`] = `
<DocumentFragment>
  <div>
    Render
  </div>
</DocumentFragment>
`;

exports[`Render tests take a snapshot test (render using other sintax) 1`] = `
<DocumentFragment>
  <div>
    Render
  </div>
</DocumentFragment>
`;

exports[`Snapshot test should take a snapshot 1`] = `
<DocumentFragment>
  <div>
    Render
  </div>
</DocumentFragment>
`;

exports[`Snapshot test take a snapshot test (render using other sintax) 1`] = `
<DocumentFragment>
  <div>
    Render
  </div>
</DocumentFragment>
`;

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <div>
    Render
  </div>
</DocumentFragment>
`;
